tests/integration/test_vid_club.py

```diff
@@ -64,27 +101,80 @@ def test_get_response_wrong_source():
 @pytest.mark.parametrize("source", ["jobs", "company", "product", "survey"])
 @pytest.mark.proper
 def test_get_response_sources(mock_api_response, source):
-    vc = VidClub()
     query = vc.get_response(source=source, to_date="2023-03-24", from_date="2023-03-24")
 
     assert isinstance(query, pd.DataFrame)
+    mock_api_response.assert_called_once()
 
 
 @pytest.mark.proper
 def test_get_response_wrong_date():
     with pytest.raises(
-        ValidationError, match=r"from_date cannot be earlier than 2023-03-22!!"
+        ValidationError, match=r"from_date cannot be earlier than 2022-03-22"
     ):
-        vc = VidClub()
         query = vc.get_response(source="jobs", from_date="2021-05-09")
```

Comment:
Please add docstring to test function that will shortly summarize what they are testing